fake_switches/cisco/command_processor/config.py

```diff
@@ -48,13 +48,20 @@ def do_no_vlan(self, *args):
         if vlan:
             self.switch_configuration.remove_vlan(vlan)
 
-    def do_no_ip(self, _, name):
-        self.switch_configuration.remove_vrf(name)
+    def do_no_ip(self, cmd, *args):
+        if "vrf".startswith(cmd):
+            self.switch_configuration.remove_vrf(args[0])
+        elif "route".startswith(cmd):
+            self.switch_configuration.remove_static_route(args[0], args[1])
 
-    def do_ip(self, cmd, name, *_):
-        vrf = self.switch_configuration.new("VRF", name)
-        self.switch_configuration.add_vrf(vrf)
-        self.move_to(ConfigVRFCommandProcessor, vrf)
+    def do_ip(self, cmd, *args):
+        if "vrf".startswith(cmd):
+            vrf = self.switch_configuration.new("VRF", args[0])
+            self.switch_configuration.add_vrf(vrf)
+            self.move_to(ConfigVRFCommandProcessor, vrf)
+        elif "route".startswith(cmd):
+            static_route = self.switch_configuration.new("Route", *args)
+            self.switch_configuration.add_static_route(static_route)
```

Comment:
Not sure about brocade... but Cisco throws an error if the destination IP is NOT a network ip (based on the mask)... I don't see that anywhere in the code for now?